meetups/meetup-diy-kit.md

```diff
@@ -57,17 +57,23 @@ Below we've linked to several guides that we put together to help you with plann
 
 ## Acquiring Funds
 
-The Community Development group has allocated funds to help members of our community do things such as hosting local meetups. Members are encouraged to do stand-alone meetups as they are an important and fun part of the growth of MakerDAO.
-
 Submitting a completed [Community Meetup Form](https://airtable.com/shr4HOtcZ8o3VZmek) is equivalent to applying for a micro-grant. Upon submitting this form, we will reach out to you through the associated email address that you used. The volume of submissions may be high, please allow up to a week of processing time before you attempt to follow up on a submission.
 
+### Who can apply for a micro-grant?
+The Community Development Group has allocated these funds to help members of our community host their own local meetups. If you're a community member who is passionate about MakerDAO and Defi, this form is for you. 
+
+### Who should not apply for a micro-grant?
+
+If you are an event organizer seeking funding for a large event, this form is not for you. 
+We are not funding xDai/Burner Wallet happy hours. 
```

Comment:
I think it might be clearer to say that we aren't funding alcohol/bar tabs or simple airdrops of Dai or xDai.